Makefile

```diff
@@ -1,4 +1,6 @@
 OPERATOR_IMAGE ?= storageos/cluster-operator:test
+# Image URL for manifest image.
+MANIFESTS_IMG ?= storageos/operator-manifests:cluster-test
```

Comment:
We use `test` in most places - any reason for `cluster-test`?

Reply:
New operator uses the same docker repo (as you suggested), and we are not able to use the same tag.

Reply:
If you create a separated repo for cluster operator manifests, i'm glad to use it.

Reply:
makes sense - thanks